airflow/www/views.py

```diff
@@ -1602,7 +1602,7 @@ def trigger(self, session=None):
                     'airflow/trigger.html', dag_id=dag_id, origin=origin, conf=request_conf, form=form
                 )
 
-        dag = current_app.dag_bag.get_dag(dag_id)
+        dag: DAG = current_app.dag_bag.get_dag(dag_id)
```

Comment:
```suggestion
        dag = current_app.dag_bag.get_dag(dag_id)
```

Revert this as we don't otherwise touch this file.